packages/experimental-utils/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
`types` should be pointing to `index.d.ts` otherwise Intellij is having issues with locating it 

we are doing this for other packages